src/platform/mhv/api/mocks/shared/maintenance-windows/index.js

```diff
@@ -0,0 +1,51 @@
+const moment = require('moment');
```

Comment:
Could we use `date-fns` instead, given momentjs was deprecated a few years ago? I did a similar thing in [src/platform/mhv/downtime/mocks/api/maintenance-windows/index.js](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/platform/mhv/downtime/mocks/api/maintenance-windows/index.js), by the way. 

I probably should have put this mocks where this code is now, so thanks for putting these changes up